eth/fetcher/block_fetcher_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/consensus/beacon"
```

Comment:
Shouldn't these changes also be reverted? Since we don't have withdrawals and block fetchers together, seems weird to add it to the tests.